clj/src/slipstream/ui/models/vms.clj

```diff
@@ -11,7 +11,7 @@
                     :measurement])
       (assoc        :username           (-> vm-metadata :user       not-empty))
       (assoc        :cloud-instance-id  (-> vm-metadata :instanceId not-empty))
-      (assoc        :run-uuid           (-> vm-metadata :runUuid    (s/replace "Unknown" "") not-empty))
```

Comment:
Currently the SlipStream Server will never send a `runUuid` field with `Unknown` as value.
See http://ssapi.sixsq.com/#virtual-machines.
So I'm asking if this `(s/replace "Unknown" "")` is needed.